Assets/Drag.cs

```diff
@@ -35,11 +64,35 @@ private void StoppedMotion(ManipulationEventData arg0)
         {
             Destroy(gameObject); // delete the extra block
         }
-
+        else
+        {
+            transform.parent = console.transform;
+            gameObject.GetComponent<Drag>().enabled = false;
+            enableFlag = false;
```

Comment:
Is this `enableFlag` to keep track of the `Drag` enabled state? If so, you should just use the `GetComponent<Drag>().enabled` in its place, this keeps state in a single variable as opposed to two places